jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/SecuredRegistrationEndpoint.java

```diff
@@ -54,4 +66,43 @@ public Response registerSecured(@DefaultValue("") @HeaderParam("x-ag-old-token")
 		return register(var, oldToken, entity, synchronously, request);
 	}
 
+	/**
+	 * Endpoint for adding UTR to an existing user at Keycloak.
+	 * Notice: after successful UTR sync user credentials shall be disabled
+	 *
+	 * @param alias User alias
+	 * @param request The request object
+	 *
+	 * return UTR
+	 *
+	 * @statuscode 200 Successful storage of the alias.
+	 * @statuscode 404 if keyclaok user or UTR for the user not found
+	 *
+	 */
+	@POST
+	@Path("/syncUtr/{alias}")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response syncUtr(@PathParam("alias") String alias, @Context HttpServletRequest request) {
```

Comment:
Why do we need to receive `alias` here? 
Can't we get the alias of of the authenticated user info we get from the JWT passed through the bearer?